CalculatorTest.py

```diff
@@ -20,6 +20,9 @@ def test_run_operation_divide(self):
         with self.assertRaises(ZeroDivisionError):
             Calculator.run_operation(9, 0, '/')
 
+    def test_run_operation_modulo(self):
+        self.assertEqual(7, Calculator.run_operation(7, 10, '%'))
```

Comment:
Good test case, maybe and 2 more, with two negative numbers and a positive and negative number